server/src/main/java/org/apache/iotdb/db/metadata/mtree/traverser/Traverser.java

```diff
@@ -106,15 +112,21 @@ protected IMNode getChild(IMNode parent, String childName) throws MetadataExcept
     if (parent.isAboveDatabase()) {
       child = parent.getChild(childName);
     } else {
-      if (parent.getSchemaTemplateId() != NON_TEMPLATE) {
-        if (!skipPreDeletedSchema || !parent.getAsEntityMNode().isPreDeactivateTemplate()) {
+      child = store.getChild(parent, childName);
```

Comment:
Perhaps still trying to fetch from the template first and then calling store.getChild would reduce some disk I/O in schemaFile mode. Just need to avoid the NPE when fetching from the template.

Reply:
Fixed.